framework/utils.py

```diff
@@ -32,6 +38,36 @@ def secure_filename(filename):
     return secure
 
 
+_sentinel: Final = object()
+
+
+def sanitize_html(
+    text: str,
+    tags: Iterable[str] = bleach.ALLOWED_TAGS,
```

Comment:
This might be fine, but according to the docs[1] the default is `bleach.sanitizer.ALLOWED_TAGS`. Not sure if they also export `ALLOWED_TAGS` under the `bleach` namespace.  The same is true for `ALLOWED_PROTOCOLS` below.

https://bleach.readthedocs.io/en/latest/clean.html#bleach.clean